docs/en/Configuration-Settings.md

```diff
@@ -81,9 +81,27 @@ $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
 Make sure that this file is distributed to `${ALLUXIO_HOME}/conf` on every Alluxio node (masters
 and workers) before starting the cluster.
 
-> Note that, `${ALLUXIO_HOME}/conf/alluxio-site.properties` are only loaded by Alluxio server
-> processes. Applications interacting with Alluxio service through Alluxio client will not load
-> these site property files, unless `${ALLUXIO_HOME}/conf` is on the application classpath.
+## Use Cluster Default
+
+Since v1.8, each Alluxio client can initialize the configuration with cluster-wise
```

Comment:
I think the uses of cluster-wise should say cluster-wide

Reply:
oops. fixed